client/store.js

```diff
@@ -174,14 +174,18 @@ function sortOperations(ops) {
             return 1;
         if (ad > bd)
             return -1;
-
-        let at = a.title,
-            bt = b.title;
-        if (at > bt)
-            return -1;
-        if (at < bt)
-            return 1;
-        return 0;
+        let ac,bc;
+        if (a.customLabel === null || a.customLabel.replace(/ /g,'').length == 0) {
```

Comment:
hint: never use `==`, but always `===`, to avoid implicit JS coercions. Again, instead of a weird regexp, you can just test `a.customLabel.trim().length` to remove spaces ;)